### PR TITLE
Individual pages, Categories, Category, Favorites pages and Adjust modal styles

### PR DESCRIPTION
**Pull request by retif**
https://3.basecamp.com/3608022/buckets/17045978/todos/3182757405
https://3.basecamp.com/3608022/buckets/17045978/todos/3195268180

## Steps to test:

- [ ] Open Video page, Stream page, Virtual Meeting page, Blog page
- [ ] Check Categories Page from Top Menu
- [ ] Check Favorites Page from Top Menu
- [ ] Check Category Page
- [ ] Check Adjust modal windows on Category Page, Videos Page, Blogs Page, Categories Page, Category Page, Favorites Page
- [ ] Check styles

## Quality checks:

Please check these boxes to confirm this PR covers the following cases:

- Maintaining our upgrade path is essential. Check one or the other:
  - [ ] This PR  provides updates via `hook_update_N` or other means.
  - [ ] No updates are necessary for this change.
- Front end fixes should be tested against all of the Open Y Themes.
  - [ ] Tested against Carnation
  - [ ] Tested against Lily
  - [ ] Tested against Rose
  - [ ] This change does not contain front-end fixes.
- [ ] I have flagged this PR "Needs Review" or pinged the VY devs/QA team in Slack

### CODE REVIEW

**Comment by retif** [comment]
retest

File: js/gated-content/src/scss/components/_add-to-calendar.scss [hunk 1]
```diff
@@ -1,4 +1,17 @@
 .add-to-calendar {
+  button {
+    margin-right:     10px;
+    outline:          none;
+    border:           2px solid #231f20;
```
Comment (anpolimus) [inline js/gated-content/src/scss/components/_add-to-calendar.scss:5]: Please, move to variables

File: js/gated-content/src/scss/components/_add-to-calendar.scss [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+    border-radius:    50%;
+    background-color: white;
```
Comment (anpolimus) [inline js/gated-content/src/scss/components/_add-to-calendar.scss:7]:
```suggestion
    background-color: $white;
```

File: js/gated-content/src/scss/utilities/_spacing.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
+.pt-20-10 {
+  @include median(padding-top, 20px, 10px);
+}
+
+.pb-20-10 {
+  @include median(padding-bottom, 20px, 10px);
+}
+
```
Comment (anpolimus) [inline js/gated-content/src/scss/utilities/_spacing.scss:8]:
```suggestion
// padding y axis, 20 for desktop, 10 for mobile and tablet
```

File: js/gated-content/src/scss/views/_video-page.scss [hunk 4]
```diff
@@ -1,127 +1,95 @@
...
-  .video {
-    padding-top: 0;
-    padding-bottom: 0;
-
-    @media all and (max-width: 768px) {
-      padding: 0;
-      width: 100%;
-    }
+  .video-wrapper {
+    background-color: black;
```
Comment (anpolimus) [inline js/gated-content/src/scss/views/_video-page.scss:5]:
```suggestion
    background-color: $black;
```

File: js/gated-content/src/scss/views/_video-page.scss [hunk 5]
```diff
@@ -1,127 +1,95 @@
...
+            content: ', ';
+          }
 
-    span+span::before {
-      content: ', ';
-      display: inline;
-    }
-  }
+          a {
+            color: white;
```
Comment (anpolimus) [inline js/gated-content/src/scss/views/_video-page.scss:74]:
```suggestion
            color: $white;
```

**Comment by retif** [comment]
retest

File: assets/img/icons/Adjust Icon.svg [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18">
+  <path id="Adjust_Icon" data-name="Adjust Icon" d="M3,17v2H9V17ZM3,5V7H13V5ZM13,21V19h8V17H13V15H11v6ZM7,9v2H3v2H7v2H9V9Zm14,4V11H11v2ZM15,9h2V7h4V5H17V3H15Z" transform="translate(-3 -3)" fill="#636466"/>
+</svg>
```
Comment (NightHunterSV) [inline assets/img/icons/Adjust Icon.svg:3]: Please, rename to avoid spaces in a file name

**Comment by NadyaRudenko** [comment]
@retif when I add a video to favorites, the icon doesn't change the color to red. here is the link to the recorded video https://monosnap.com/file/C1BABDuLMYKbzxOM4LCTbyRhBACPc3
please change the order for the info displayed on the individual video page, live stream, virtual meeting pages 
image
need the confirmation on the category icon for the blog page
image
in design it is absent. but for the rest content indication of category looks different
image

**Comment by NadyaRudenko** [comment]
@retif no action for the category on the blog page. leave it as it is.

**Comment by retif** [comment]
retest

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -20,4 +20,5 @@ Please check these boxes to confirm this PR covers the following cases:
   - [ ] Tested against Lily
   - [ ] Tested against Rose
   - [ ] This change does not contain front-end fixes.
-- [ ] I have flagged this PR "Needs Review" or pinged the VY devs/QA team in Slack
+- [ ] I have flagged this PR "Needs Review" or pinged the VY devs/QA
```
Comment (anpolimus) [inline .github/pull_request_template.md:23]: 👍 👍 👍 👍 👍

**Review by anpolimus — APPROVED** [review]

**Comment by NadyaRudenko** [comment]
@retif please fix the following:

- font-family for filters titles on modal window (Rose and Carnation themes)
image

- modal styles for Lily and Rose themes
IMG_F0B69CF89AD6-1
IMG_8BDA0433FF8E-1

- color of radio button to green 
image

**Comment by NadyaRudenko** [comment]
@retif can you please place items in modal window in 2 rows for the tablet?
design https://xd.adobe.com/view/e9d9a7ca-1747-4b26-6f69-79dcd76ecadf-9fe8/screen/0d23f4dd-75aa-4bdd-bb5c-690e1b3b55b3/
image
Thank you!

**Comment by NadyaRudenko** [comment]
retest

**Comment by NadyaRudenko** [comment]
@retif please review the following:

- button border color on Carnation
image

- Lily, Rose font-weight for filters
image

- color for all themes
image

- Buttons
image
image
image

**Review by hamrant — APPROVED** [review]

**Review by hamrant — APPROVED** [review]

**Review by anpolimus — DISMISSED** [review]

**Comment by retif** [comment]
retest

**Review by NadyaRudenko — APPROVED** [review]
I'm approving this crazy long PR
great work @retif! thank you very much! 🪐⭐️☄️🔥

**Review by hamrant — APPROVED** [review]

**Pull request merged**